app/models/concerns/ats/ashbyhq/job_details.rb

```diff
@@ -18,7 +18,7 @@ def job_url_api(url_api, ats_identifier, _job_id)
         "#{url_api}#{ats_identifier}?includeCompensation=true"
       end
 
-      def fetch_url(job_data)
+      def fetch_url(job_data, _company_id)
```

Comment:
What is _company_id doing here?

Reply:
Nothing useful! It was a lazy solution to the problem that only BambooHR needs info extrinsic to the all_jobs json in order to build the posting_url, but I've found a workaround and removed it.